File: source/api/cards/update/_update.java
Customer customer = Customer.retrieve("cust_test_4xsjvylia03ur542vn6");
Card card = customer.getCards().retrieve("card_test_4xsjw0t21xaxnuzi9gs");
card.update(new HashMap<String, Object>() {
  {put("expiration_month", 11);}
  {put("expiration_year", 2017);}
  {put("name", "Somchai Praset");}
  {put("postal_code", "10310");}
});

File: source/v20140727/api/cards/update/_update.cs
var card = new CardUpdateInfo ();
card.Id = "card_test_4xsjw0t21xaxnuzi9gs";
card.Name = "Somchai Praset";
card.ExpirationMonth = 11;
card.ExpirationYear = 2017;

var updateResult = client.CardService.UpdateCard ("cust_test_4xsjvylia03ur542vn6", card);
